code/modules/projectiles/projectile/plasma.dm

```diff
@@ -106,6 +106,11 @@
 
 	fire_stacks = TRUE
 
+/obj/item/projectile/plasma/aoe/heat/strong/mech
```

Comment:
I would suggest also increasing the AoE damage by a bit. No need to increase the radius, however.